.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "jpo-sdw-depositor"]
 	path = jpo-sdw-depositor
 	url = https://github.com/usdot-jpo-ode/jpo-sdw-depositor.git
+[submodule "qa/test-harness/ode-output-validator-library"]
```

Comment:
Lets not include this as a submodule if possible. This is not very portable.

See https://github.com/usdot-jpo-ode/ode-output-validator-library/pull/15 for info about how to add the file to the library.

Reply:
Will do at a later time.